src/DotNetWorker.Grpc/GrpcModelBindingData.cs

```diff
@@ -5,20 +5,23 @@
 
 namespace Microsoft.Azure.Functions.Worker.Grpc.Messages
 {
-    internal partial class GrpcModelBindingData : Microsoft.Azure.Functions.Worker.Core.ModelBindingData
+    internal class GrpcModelBindingData : Microsoft.Azure.Functions.Worker.Core.ModelBindingData
     {
         private readonly ModelBindingData _modelBindingData;
 
+        private readonly BinaryData _content;
```

Comment:
Do we need this private var? `Content` prop value can be set inside the constructor.